src/detect-byte.h

```diff
@@ -27,6 +27,6 @@
 
 typedef uint8_t DetectByteIndexType;
 
-bool DetectByteRetrieveSMVar(const char *, const Signature *, int sm_list, DetectByteIndexType *);
-
+bool DetectByteRetrieveSMVar(const char *, const Signature *, bool strict, int sm_list,
+        DetectByteIndexType *, const DetectEngineCtx *);
```

Comment:
@jasonish what is our policy for a patch release ?
This is ok as it is not a "public" API with SC prefix ?

Reply:
Public API is more about documentation than the SC prefix. This only does not have the SC prefix as it hasn't been done yet.  The SC prefix is more about preventing name clashes with other libraries than designating functions part of a public API.

It is a breaking change though. What are the chances it would affect anyone?

Reply:
Jeff, this is not about the rule writer here, it is about the plugin writer.

If some plugin (or library user) uses `DetectByteRetrieveSMVar`, this is a breaking change.

> What are the chances it would affect anyone?

I do not know if a plugin or library uses `DetectByteRetrieveSMVar` (or other changed prototypes)